cpanfile

```diff
@@ -0,0 +1,14 @@
+requires "Class::Method::Modifiers" => "0";
+requires "Email::Stuffer" => "0";
+requires "Mojo::UserAgent" => "0";
```

Comment:
I recommend specifying a particular release of Mojolicious here since you use both Mojo::DOM and Mojo::UserAgent:

```perl
requires 'Mojolicious', '6.00';
```

Reply:
I'm not familiar with Mojolicious versions but I have used your suggestion